## glibc-2.8-port.diff by  <sonicmctails@gmail.com>
##
## Corrects FTBFS due to glibc 2.8

diff -urNad tslib-1.0~/tests/ts_calibrate.c tslib-1.0/tests/ts_calibrate.c
--- tslib-1.0~/tests/ts_calibrate.c	2008-11-18 17:50:32.000000000 -0500
+++ tslib-1.0/tests/ts_calibrate.c	2008-11-18 17:52:01.000000000 -0500
@@ -17,6 +17,7 @@
 #include <signal.h>
 #include <string.h>
 #include <unistd.h>
+#include <sys/stat.h>
 #include <sys/fcntl.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
@@ -224,9 +225,9 @@
 		for (i = 0; i < 7; i++) printf("%d ", cal.a [i]);
 		printf("\n");
 		if ((calfile = getenv("TSLIB_CALIBFILE")) != NULL) {
-			cal_fd = open (calfile, O_CREAT | O_RDWR);
+			cal_fd = open (calfile, O_CREAT | O_RDWR, S_IRWXU | S_IRWXG | S_IRWXO);
 		} else {
-			cal_fd = open ("/etc/pointercal", O_CREAT | O_RDWR);
+			cal_fd = open ("/etc/pointercal", O_CREAT | O_RDWR, S_IRWXU | S_IRWXG | S_IRWXO);
 		}
 		sprintf (cal_buffer,"%d %d %d %d %d %d %d",
 			 cal.a[1], cal.a[2], cal.a[0],
